src/tests/intg/test_infopipe.py

```diff
@@ -824,3 +834,22 @@ def test_list_by_attr(dbus_system_bus, ldap_conn, sanity_rfc2307):
 
     users = get_user_by_attr(dbus_system_bus, "noattr", "*")
     assert len(users) == 0
+
+
+def test_list_by_name(dbus_system_bus, ldap_conn, sanity_rfc2307):
+    users = get_user_by_name(dbus_system_bus, "user2")
```

Comment:
Would it make sense to also test group-by-name (`infopipe.Groups`)?

Reply:
FTR, I just found that `Groups.ListByName()` and `Groups.ListByDomainAndName()` do not work and didn't work before these changes. I'll open a ticket.

Reply:
> FTR, I just found that `Groups.ListByName()` and `Groups.ListByDomainAndName()` do not work and didn't work before these changes. I'll open a ticket.

Do you plan to include this fix in this PR or do you prefer to leave it out for a different PR?

Reply:
I think it will be better to keep it in a separate PR.

Reply:
Then I think this separate PR should be merged before this one, to make sure patches here don't introduce another issue in `Groups.FindByName/ListByDomainAndName`.

Reply:
I created the review for that fix: https://github.com/SSSD/sssd/pull/6406